src/Compiler/Driver/CompilerDiagnostics.fs

```diff
@@ -2114,6 +2115,10 @@ let FormatDiagnosticLocation (tcConfig: TcConfig) m : FormattedDiagnosticLocatio
                     sprintf "%s(%d,%d,%d,%d): " file m.StartLine m.StartColumn m.EndLine m.EndColumn, m, file
                 else
                     "", m, file
+            | DiagnosticStyle.Rich ->
+                let file = file.Replace('/', Path.DirectorySeparatorChar)
+                let m = withStart (mkPos m.StartLine (m.StartColumn + 1)) m
+                (sprintf "◦→ %s (%d,%d)" file m.StartLine m.StartColumn), m, file
```

Comment:
I am not an expert in any encoding stuff, random question: can these symbols accidentally produce some display issues anywhere? Why not just use good old `->`? :)

Reply:
Probably they do. That's why I'd like to merge it as is, and then have couple of PRs with different formats to see which one works/easier to consume

Reply:
Reasoning is that it's strictly opt-in with a hidden flag, not known to anyone

Reply:
Alright, I was just curious why this was even picked in the first place.